src/wp-includes/cache.php

```diff
@@ -145,6 +145,63 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 	return $wp_object_cache->get_multiple( $keys, $group, $force );
 }
 
+
+/**
+ * Retrieves multiple values from the cache in one call.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::delete_multiple()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param array  $keys  Array of keys under which the cache to deleted.
+ * @param string $group Optional. Where the cache contents are grouped. Default empty.
+ * @return array Array of return values organized into groups.
+ */
+function wp_cache_delete_multiple( $keys, $group = '' ) {
+	global $wp_object_cache;
+
+	return $wp_object_cache->delete_multiple( $keys, $group );
+}
+
+/**
+ * Add multiple values from the cache in one call.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::add_multiple()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param array  $data  Array of key and value to be set.
+ * @param string $group Optional. Where the cache contents are grouped. Default empty.
+ * @param int    $expire Optional. When to expire the cache contents, in seconds.
+ *                           Default 0 (no expiration).
+ * @return array Array of return values organized into groups.
+ */
+function wp_cache_add_multiple( $data, $group = '', $expire = 0 ) {
+	global $wp_object_cache;
+
+	return $wp_object_cache->add_multiple( $data, $group, $expire );
+}
+
+/**
+ * Set multiple values from the cache in one call.
+ *
+ * @see WP_Object_Cache::set_multiple()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param array  $data  Array of key and value to be set.
+ * @param string $group Optional. Where the cache contents are grouped. Default empty.
+ * @param int    $expire Optional. When to expire the cache contents, in seconds.
+ *                           Default 0 (no expiration).
+ * @return array Array of return values organized into groups.
+ */
+function wp_cache_set_multiple( $data, $group = '', $expire = 0 ) {
+	global $wp_object_cache;
+
+	return $wp_object_cache->set_multiple( $data, $group, $expire );
+}
+
```

Comment:
Do we need `wp_cache_replace_multiple()` as well?

Reply:
The wp_cache_replace function is not used in core anywhere, so I am not sure of it's value as part of this patch.

Reply:
> The wp_cache_replace function is not used in core anywhere, so I am not sure of it's value as part of this patch.

Guess what... 😅